src/js/components/ManualBreadcrumbs.js

```diff
@@ -0,0 +1,210 @@
+import classNames from 'classnames';
+import DeepEqual from 'deep-equal';
+import React, {PropTypes} from 'react';
+import ReactDOM from 'react-dom';
+
+import BreadcrumbSegmentLink from './BreadcrumbSegmentLink';
+import IconChevron from './icons/IconChevron';
+
+const COLLAPSE_BUFFER = 12;
+const LAST_ITEM_OFFSET = 150; // Difference between scrollWidth and outerWidth
+const PADDED_ICON_WIDTH = 38; // Width of icon + padding
+
+class ManualBreadcrumbs extends React.Component {
```

Comment:
I'd like rename this one to just `Breadcrumbs` and the other to `RoutedBreadcrumbs`  as this one will serve as the base class for all breadcrumbs. What do you think?